src/algorithms/configuration/maps/dense_map.py

```diff
@@ -68,43 +64,28 @@ def extend_walls(self) -> None:
         Method for extending the walls by agent radius
         """
 
-        #def extend_obstacle_bound() -> None:
-        #    for b in bounds:
-        #        for x in range(b.x - self.agent.radius, b.x + self.agent.radius + 1):
-        #            for y in range(b.y - self.agent.radius, b.y + self.agent.radius + 1):
-        #                if not self.is_out_of_bounds_pos(Point(x, y)):
-        #                    dist: Union[float, np.ndarray] = np.linalg.norm(np.array([x, y]) - np.array(b))
-        #                    if dist <= self.agent.radius and self.grid[y][x] == DenseMap.CLEAR_ID:
-        #                        self.grid[y][x] = DenseMap.EXTENDED_WALL
-        #                        self.obstacles.append(ExtendedWall(Point(x, y)))
-        #                        visited.add(Point(x, y))
-
         def extend_obstacle_bound() -> None:
             for b in bounds:
-                for x in range(b.x - self.agent.radius, b.x + self.agent.radius + 1):
-                    for y in range(b.y - self.agent.radius, b.y + self.agent.radius + 1):
-                        if not self.is_out_of_bounds_pos(Point(x, y)):
-                            dist: Union[float, np.ndarray] = np.linalg.norm(np.array([x, y]) - np.array(b))
-                            if dist <= self.agent.radius and self.grid[x][y] == DenseMap.CLEAR_ID:
-                                self.grid[x][y] = DenseMap.EXTENDED_WALL
-                                self.obstacles.append(ExtendedWall(Point(x, y)))
-                                visited.add(Point(x, y))
+                for index in np.ndindex(*(len(self.size) * [self.agent.radius + 1])):
+                    for sign in [-1, 1]:
```

Comment:
I think this is wrong, this would check (-1, -1) and (1, 1) but not (-1, 1) etc.